assets/requirements.txt

```diff
@@ -0,0 +1,2 @@
+dpath
```

Comment:
dpath and requests not used in code base, so no need to have them in the import/requirements.